<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InputKeyboard_Window</class>
 <widget class="QMainWindow" name="InputKeyboard_Window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>700</width>
    <height>400</height>
   </size>
  </property>
  <property name="sizeIncrement">
   <size>
    <width>700</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget_Mark2" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-30</y>
      <width>701</width>
      <height>461</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget
{
background: url(&quot;://rec//img//Background.png&quot;);
background-repeat: repeat;

  height: 500px; /* You must set a specified height */
  background-position: center; /* Center the image */
  background-repeat: no-repeat; /* Do not repeat the image */
  background-size: cover; /* Resize the background image to cover the entire container */
}

</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>80</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background: transparent;</string>
     </property>
     <property name="text">
      <string>Surname:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>130</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background: transparent;</string>
     </property>
     <property name="text">
      <string>Name:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>180</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background: transparent;</string>
     </property>
     <property name="text">
      <string>Data of birth:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>230</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background: transparent;</string>
     </property>
     <property name="text">
      <string>Marks:</string>
     </property>
    </widget>
    <widget class="QDateEdit" name="dateEdit">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>190</y>
       <width>110</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: white;</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_Mark1">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>240</y>
       <width>42</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: white;</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_Mark2">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>240</y>
       <width>42</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: white;</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_Mark3">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>240</y>
       <width>42</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: white;</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_Mark4">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>240</y>
       <width>42</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: white;</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_Mark5">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>240</y>
       <width>42</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: white;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_OK">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>300</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton#pushButton_OK {

background: url(&quot;://rec//img//btn_OK.png&quot;);
 	background-repeat: repeat-none;
	background-position: middle;
	border-style: outset;
    border-radius: 10px;
    border-color: beige;
    font: bold 14px;
    min-width: 2em;
    padding: 6px;

}

QPushButton#pushButton_OK:hover {

background: url(&quot;://rec//img//btn_OK_hover.png&quot;);
	background-repeat: repeat-none;
    border-style: inset;
	background-position: middle;
 }

QPushButton#pushButton_OK:pressed {

background: url(&quot;://rec//img//btn_OK_pressed.png&quot;);
	background-repeat: repeat-none;
	background-position: bottom;

}

</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_Cancel">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>300</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton#pushButton_Cancel {

background: url(&quot;://rec//img//btn_Cancel.png&quot;);
 	background-repeat: repeat-none;
	background-position: middle;
	border-style: outset;
    border-radius: 10px;
    border-color: beige;
    font: bold 14px;
    min-width: 2em;
    padding: 6px;

}

QPushButton#pushButton_Cancel:hover {
background: url(&quot;://rec//img//btn_Cancel_hover.png&quot;);
	background-repeat: repeat-none;
    border-style: inset;
	background-position: middle;
 }

QPushButton#pushButton_Cancel:pressed {

background: url(&quot;://rec//img//btn_Cancel_pressed.png&quot;);
	background-repeat: repeat-none;
	background-position: bottom;

}

</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_Surname">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>90</y>
       <width>151</width>
       <height>20</height>
      </rect>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::ActionsContextMenu</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">background: white;</string>
     </property>
     <property name="inputMethodHints">
      <set>Qt::ImhDigitsOnly</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_Name">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>140</y>
       <width>151</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: white;</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>700</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
